mmdet/apis/ote/apis/detection/configuration.py

```diff
@@ -146,9 +156,10 @@ class __POTParameter(ParameterGroup):
 
         preset = selectable(default_value=POTQuantizationPreset.PERFORMANCE, header="Preset",
                             description="Quantization preset that defines quantization scheme",
-                            editable=False, visible_in_ui=False)
+                            editable=True, visible_in_ui=True)
```

Comment:
Have you run tests? configuration.py does not correspond to configuration.yaml (editable, visible_in_ui)

Reply:
Fixed